public/content/contributing/adding-resources/index.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Adding Resources
+description: The policy we use when adding resources to ethereum.org
+lang: en
+---
+
+# Adding Resources {#adding-resources}
+
+We want to make sure we list the best resources possible while keeping users safe and confident.
+
+Anyone is free to suggest new resources to add to the resource dashboard on ethereum.org, currently found at [ethereum.org/resources](/resources/).
```

Comment:
@nloureiro any thoughts on what we should call this page? resources doesnt seem right, but @wackerow mentioned that he feels the same about ethereum.org 'dashboard'

Reply:
Yeah, would suggest perhaps a redirect from `/dashboards/` -301-> `/resources/` if we want to use that